optimum/exporters/onnx/model_configs.py

```diff
@@ -256,6 +258,27 @@ class ImageGPTOnnxConfig(GPT2OnnxConfig):
     pass
 
 
+class DecisionTransformerOnnxConfig(GPT2OnnxConfig):
+    DUMMY_INPUT_GENERATOR_CLASSES = (
+        DummyDecisionTransformerInputGenerator,
+    )
+
+    @property
+    def inputs(self) -> Dict[str, Dict[int, str]]:
+        DEFAULT_DUMMY_SHAPES['actions'] = self._normalized_config.config.act_dim
+        DEFAULT_DUMMY_SHAPES['states'] = self._normalized_config.config.state_dim
+        DEFAULT_DUMMY_SHAPES['returns'] = 1
+        DEFAULT_DUMMY_SHAPES['last_hidden_state'] = self._normalized_config.config.hidden_size
+
+        return {
+            'states': {0: 'batch_size', 1: 'sequence_length', 2: 'states'},
+            'actions': {0: 'batch_size', 1: 'sequence_length', 2: 'actions'},
+            'returns_to_go': {0: 'batch_size', 1: 'sequence_length', 2: 'returns'},
+            'timesteps': {0: 'batch_size', 1: 'sequence_length'},
+            'attention_mask': {0: 'batch_size', 1: 'sequence_length'},
```

Comment:
use original dimensions names names here as well (`state_dim` and `act_dim` and `max_ep_len`)
naming them states, actions and returns is confusing because those are the names of the output tensors themselves.